### PR TITLE
Update submit buttons to use `.jenkins-button` classes

### PR DESCRIPTION
**Pull request by janfaracik**
Small PR which updates the submit buttons to use the `.jenkins-button` classes, along with @NotMyFault's work this should be the majority of buttons converted 🚀 

**Before**

image

**After**

image

### Proposed changelog entries

* Update submit buttons to use `.jenkins-button` classes.

### Proposed upgrade guidelines

N/A

### Submitter checklist

- [ ] (If applicable) Jira issue is well described
- [ ] Changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developer, depending on the change) and are in the imperative mood. [Examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)
  - Fill-in the `Proposed changelog entries` section only if there are breaking changes or other changes which may require extra steps from users during the upgrade
- [ ] Appropriate autotests or explanation to why this change has no tests
- [ ] New public classes, fields, and methods are annotated with `@Restricted` or have `@since TODO` Javadoc, as appropriate.
- [ ] New deprecations are annotated with `@Deprecated(since = "TODO")` or `@Deprecated(forRemoval = true, since = "TODO")` if applicable.
- [ ] New or substantially changed JavaScript is not defined inline and does not call `eval` to ease future introduction of Content-Security-Policy directives (see [documentation on jenkins.io](https://www.jenkins.io/doc/developer/security/csp/)).
- [ ] For dependency updates: links to external changelogs and, if possible, full diffs

### Desired reviewers

@jenkinsci/sig-ux 

### Maintainer checklist

Before the changes are marked as `ready-for-merge`:

- [ ] There are at least 2 approvals for the pull request and no outstanding requests for change
- [ ] Conversations in the pull request are over OR it is explicit that a reviewer does not block the change
- [ ] Changelog entries in the PR title and/or `Proposed changelog entries` are accurate, human-readable, and in the imperative mood
- [ ] Proper changelog labels are set so that the changelog can be generated automatically
- [ ] If the change needs additional upgrade steps from users, `upgrade-guide-needed` label is set and there is a `Proposed upgrade guidelines` section in the PR title. ([example](https://github.com/jenkinsci/jenkins/pull/4387))
- [ ] If it would make sense to backport the change to LTS, a Jira issue must exist, be a _Bug_ or _Improvement_, and be labeled as `lts-candidate` to be considered (see [query](https://issues.jenkins.io/issues/?filter=12146)).

<a href="https://gitpod.io/#https://github.com/jenkinsci/jenkins/pull/7203"></a>

### CODE REVIEW

**Comment by janfaracik** [comment]
Opened ATH here - https://github.com/jenkinsci/acceptance-test-harness/pull/960

File: core/src/main/resources/lib/form/submit.jelly
Comment (timja) [inline core/src/main/resources/lib/form/submit.jelly:28]: cdata should be restored so `<button>` can be used without escaping

**Review by timja — COMMENTED** [review]
not tested yet

File: core/src/main/resources/lib/form/apply/apply.js
Comment (yaroslavafenkin) [inline core/src/main/resources/lib/form/apply/apply.js:66]: What is this magic? 🤔

**Review by yaroslavafenkin — COMMENTED** [review]
Tested, looks OK to me in general.

Couple of oddities I've noticed, not sure if you had intent to cover those in this PR:
- ~~"OK" button on "New Item" page is not covered.~~
- Navigating to Manage -> Nodes -> New node, Save button does not have min width set.

**Comment by NotMyFault** [comment]
For the first bullet point, I'd like to mention https://github.com/jenkinsci/jenkins/pull/7191; that's not a submit button, hence addressed separately.

**Comment by yaroslavafenkin** [comment]
> For the first bullet point, I'd like to mention https://github.com/jenkinsci/jenkins/pull/7191; that's not a submit button, hence addressed separately.

And I actually reviewed that... 🤦‍♂️

File: core/src/main/resources/lib/form/apply/apply.js
Comment (janfaracik) [reply core/src/main/resources/lib/form/apply/apply.js:66]: Yeah I'm confused by it too - it was there before it's just been linted/reindented.

**Comment by janfaracik** [comment]
> * Navigating to Manage -> Nodes -> New node, Save button does not have min width set.

It does on mine?

image

File: core/src/main/resources/lib/form/apply/apply.js
Comment (yaroslavafenkin) [reply core/src/main/resources/lib/form/apply/apply.js:66]: I see indentation change, but condition inside the `if` was different
image

**Review by yaroslavafenkin — COMMENTED** [review]
> Unless you meant the edit node page

Yup, meant that one.

Re-tested, LGTM now. Still not sure about the `if` condition magic though.

File: core/src/main/resources/lib/form/apply/apply.js
Comment (timja) [reply core/src/main/resources/lib/form/apply/apply.js:66]:
```suggestion
          if (oldError) {
```

**Review by timja — APPROVED** [review]
Tested, LGTM

**Review by NotMyFault — APPROVED** [review]

File: core/src/main/resources/lib/hudson/project/makeDisabled.jelly
Comment (janfaracik) [reply core/src/main/resources/lib/hudson/project/makeDisabled.jelly:48]: Updated: 
image

Thanks!

**Comment by timja** [comment]
@yaroslavafenkin was the LGTM an approval? You didn't update the labels

or @jenkinsci/core-security-review

**Comment by yaroslavafenkin** [comment]
> @yaroslavafenkin was the LGTM an approval? You didn't update the labels
> 
> or @jenkinsci/core-security-review

At that time I still had concerns about change in `lib/form/apply.js` but it got reverted.

**Review by yaroslavafenkin — APPROVED** [review]

**Comment by timja** [comment]
/label ready-for-merge

----

This PR is now ready for merge, after ~24 hours, we will merge it if there's no negative feedback.

Thanks!

**Comment by basil** [comment]
Causes `hudson.plugins.throttleconcurrents.ThrottleQueueTaskDispatcherTest` to start failing with e.g.

```
java.lang.AssertionError: Failed to find an input with type submit on the page
        at org.junit.Assert.fail(Assert.java:89)
        at hudson.plugins.throttleconcurrents.ThrottleQueueTaskDispatcherTest.submitForm(ThrottleQueueTaskDispatcherTest.java:347)
        at hudson.plugins.throttleconcurrents.ThrottleQueueTaskDispatcherTest.configureNewNodeWithLabel(ThrottleQueueTaskDispatcherTest.java:303)
        at hudson.plugins.throttleconcurrents.ThrottleQueueTaskDispatcherTest.assertBasedOnMaxLabelPairMatchingOrNot(ThrottleQueueTaskDispatcherTest.java:178)
        at hudson.plugins.throttleconcurrents.ThrottleQueueTaskDispatcherTest.testShouldConsiderTaskAsBlockableStillUponMatchingLabelPairWithLowestMax(ThrottleQueueTaskDispatcherTest.java:123)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
        at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.base/java.lang.reflect.Method.invoke(Method.java:566)
        at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
        at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
        at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
        at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
        at org.jvnet.hudson.test.JenkinsRule$1.evaluate(JenkinsRule.java:618)
        at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:299)
        at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:293)
        at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
        at java.base/java.lang.Thread.run(Thread.java:829)
```

**Comment by daniel-beck** [comment]
Likely caused [JENKINS-70112](https://issues.jenkins.io/browse/JENKINS-70112).

**Comment by daniel-beck** [comment]
Caused [JENKINS-70662](https://issues.jenkins.io/browse/JENKINS-70662).

File: core/src/main/resources/lib/form/submit.jelly
Comment (daniel-beck) [inline core/src/main/resources/lib/form/submit.jelly:56]: Caused [JENKINS-75441](https://issues.jenkins.io/browse/JENKINS-75441)

**Pull request merged**
